graphics/include/ignition/common/SkeletonAnimation.hh

```diff
@@ -47,6 +47,11 @@ namespace ignition
       /// the animations
       public: ~SkeletonAnimation();
 
+      /// \brief Assignment operator
+      /// \param[in] _other The new SkeletonAnimation
+      /// \return A reference to this instance
+      public: SkeletonAnimation &operator=(const SkeletonAnimation &_other);
```

Comment:
this function should not be merged forward to ign-common4 right? Can you add a note as a reminder?

Reply:
Added: https://github.com/ignitionrobotics/ign-common/pull/199/commits/af8b84ad9c5be203be79fc655db07346b964213f

Let me know if there's a better way/format of adding a note like this. I don't think it should be a `TODO` since nothing really needs to be "done", and I also assume it shouldn't be a doxygen comment since this isn't relevant for user documentation (it's more of just an internal developer note).

Reply:
looks good to me.